aiida_registry/fetch_metadata.py

```diff
@@ -47,6 +50,42 @@ def get_hosted_on(url):
 
     return netloc
 
+def get_github_commits_count(repo_url):
+    """
+    Get the commits count on the default branch of the repository.
+    Maximum commits number can be retrieved is 100.
```

Comment:
I suppose with the 12 month window we are hitting this limitation for a few plugins?

Would this trick remove this limitation? https://stackoverflow.com/a/70610670/1069467

Reply:
Yes, this solved the limitation issue, thanks!